modules/sdk-coin-sui/src/lib/stakingTransaction.ts

```diff
@@ -246,4 +292,30 @@ export class StakingTransaction extends Transaction<MoveCallTx> {
       outputAmount: amount,
     };
   }
+
+  /**
+   * Returns a complete explanation for a withdraw delegation transaction
+   *
+   * @param {TxData} json The transaction data in json format
+   * @param {TransactionExplanation} explanationResult The transaction explanation to be completed
+   * @returns {TransactionExplanation}
+   */
+  explainWithdrawDelegationTransaction(
+    json: TxData,
+    explanationResult: TransactionExplanation
+  ): TransactionExplanation {
+    return {
+      ...explanationResult,
+      fee: {
+        fee: this.suiTransaction.gasBudget.toString(),
+      },
+      type: TransactionType.StakingWithdraw,
+      outputs: [
+        {
+          address: json.sender,
+          amount: '',
```

Comment:
Is there a way to get the amount?

Reply:
with the current API they have I left it blank, don't know from where to pull the staking amount, unless this amount will be supplied either by WP or Staking-service

Reply:
setting it to `TRANSFER_AMOUNT_UNKNOWN` as for other coins